dataLayer/modulemanager.datalayer.json

```diff
@@ -1,2 +1,17 @@
-{
-}
+MMdataLayer = [{
```

Comment:
Please change the single quotation marks to double quotation marks which is the standard we are using for all other data layers.

Reply:
Hi Kath - The standard dataLayer scripts in the analytics world use single quotation marks: https://developers.google.com/tag-manager/devguide. Should we keep using them for future dataLayers?

Reply:
As discussed, let's use double quotation marks here.